c/src/ml-api-inference-single.c

```diff
@@ -1083,6 +1090,11 @@ ml_single_open_custom (ml_single_h * single, ml_single_preset * info)
     status = ML_ERROR_STREAMS_PIPE;
     goto error;
   }
+  /* handle flexible single */
+  if (info->nnfw == ML_NNFW_TYPE_EXECUTORCH_LLAMA) {
```

Comment:
I think if there are more flexible filter, we may revise this part later.
The llama2c filter I created doesn't seem to be scalable due to API limitations, so I won't add it now.